src/Resources/views/form/form_gateway_config_row.html.twig

```diff
@@ -2,7 +2,15 @@
     <div class="{% if required %}required {% endif %}field{% if (not compound or force_error|default(false)) and not valid %} error{% endif %}">
         {{- form_label(form) -}}
         {{- form_widget(form) -}}
-        {{- form_help(form) -}}
+        <span style="font-size: 0.8rem; font-style: italic">
+            {{- form_help(form) -}}
+        </span>
         {{- form_errors(form) -}}
     </div>
 {% endblock %}
+
+{% block _sylius_payment_method_gatewayConfig_config_cgv_added_label -%}
+    <{{ element|default('label') }}{% if label_attr %}{% with { attr: label_attr } %}{{ block('attributes') }}{% endwith %}{% endif %}>
+        {{- label|trans(label_translation_parameters, translation_domain) | raw -}}
```

Comment:
I took some elements of `form_label`, but I needed to add the `| raw` to render the "More Info" link